app/src/main/java/org/jellyfin/androidtv/ui/playback/CustomPlaybackOverlayFragment.java

```diff
@@ -1346,14 +1284,36 @@ public void setPlayPauseActionState(final int state) {
     @Override
     public void updateDisplay() {
         BaseItemDto current = mPlaybackController.getCurrentlyPlayingItem();
-        if (current != null & mActivity != null && !mActivity.isFinishing()) {
+        if (current != null && !requireActivity().isFinishing()) {
```

Comment:
requireactivity throws an exception if activity is null, the previous if statement just did a no-op. Might cause issues.